.travis.yml

```diff
@@ -50,7 +50,7 @@ deploy:
     acl: public_read
     cache_control: "max-age=31536000, public"
     on:
-      branch: development
+      branch: push_sdk_update_on_split_kill
```

Comment:
Don't forget this baby before merging or after you're done with that canary release.